src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -173,7 +172,7 @@ function DatasetOverviewPanel(props) {
             {renderTextFieldValue('Dataset ID', dataset.id)}
           </Grid>
           <Grid item xs={4}>
-            {renderTextFieldValue('Default Billing Profile ID', dataset.defaultBillingProfile)}
+            {renderTextFieldValue('Default Billing Profile ID', dataset.defaultProfileId)}
```

Comment:
Drive-by bug fix: this was the wrong name for this field, so a dataset's default billing profile ID was always blank.

I think if this were Typescriptified we would have caught this. I tried to convert this file but it was taking longer than I wanted to, so I walked it back. I'd be interested in talking to the team about how y'all approach Typescript conversions.